### ISSUE TITLE
JS Integration in One-page apps(Vue.js) got none pageview data in last 3 days Trends page of self deploy Docker iamge

### ISSUE THREAD

**Issue by AlexShan2008** [PostHog/posthog#1246]
### Bug description
1. Since 17 July, I cannot get any pageview data from the SaaS dashboard
2. Other events data are all right, except the pageview data
3. I use `posthog.capture('$pageview')` report data when the router change every time
4. The `posthog.capture('$pageview')` http request is success

### How to reproduce

1. `npm i posthog-js -D`
2. init the posthog in main.js (Vue.js app)
3. when the router change call `posthog.capture('$pageview')` this function

### Expected behavior

I want to see the pageview data in the self-manged SaaS dashboard trend page

### Screenshots

![image](https://user-images.githubusercontent.com/23693423/87867744-c427c300-c9c2-11ea-8aa6-043edb576b26.png)

![image](https://user-images.githubusercontent.com/23693423/87867993-9cd1f580-c9c4-11ea-8550-18d70379d323.png)

![image](https://user-images.githubusercontent.com/23693423/87867784-339db280-c9c3-11ea-9dd9-460e8d078764.png)

![image](https://user-images.githubusercontent.com/23693423/87867956-5b414a80-c9c4-11ea-93fc-8b5eb9055790.png)

![image](https://user-images.githubusercontent.com/23693423/87867866-c3436100-c9c3-11ea-9d69-2fa519882a06.png)

![image](https://user-images.githubusercontent.com/23693423/87867887-f84fb380-c9c3-11ea-91c0-a1c30cc37c71.png)

### Environment

- self-managed
- NPM package: posthog-js version:1.3.4
- SPA Vue.js

**Comment by jb33k** [comment]
Yes, this is broken apparently. I also stopped getting any pageview events.

**Comment by timgl** [comment]
Thanks for raising this, are you getting any error messages? Is PostHog not doing any requests?

@Twixes could you see if you can reproduce this

**Comment by jb33k** [comment]
No error messages from the server or from `capture()`.

When posthog initializes, it makes a request to `selfhostedposthog.com/decide/?ip=1` and gets the following response...
```
{"config": {"enable_collect_everything": true}, "editorParams": {}, "isAuthenticated": false, "supportedCompression": ["gzip", "lz64"], "featureFlags": []}
```
No more requests after that.

**Comment by timgl** [comment]
Hmm, how are you using the js integration? Via snippet or npm?

**Comment by jb33k** [comment]
Snippet. Copy/paste straight from Setup view.

**Comment by timgl** [comment]
@jb33k we're struggling to reproduce this. Could you share
- the snippet code
- How you're calling posthog.capture
- Ideally the URL where you're using PostHog. Feel free to email tim@posthog.com if you don't want to share publicly

Thanks!

**Comment by jb33k** [comment]
Unfortunately I can't share the URL because it's a local dev environment (http://127.0.0.1:8000) for a project that's not available in public yet.

The snippet code is:
```
!function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.async=!0,p.src=s.api_host+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled onFeatureFlags".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
  posthog.init('XXX', {api_host: 'https://foobar.com'})
```

We pulled the latest Docker containers few hours ago and restarted everything, didn't help. `web` container isn't giving any errors etc.

**Comment by timgl** [comment]
@jb33k and @AlexShan2008 did either/both of you have the DNT in your browser set to true (1)?

**Comment by jb33k** [comment]
That was it, do not track was causing tracking to stop. 

As a sidenote, a big thank you for doing what DNT asks you to do :)

**Comment by timgl** [comment]
@jb33k Great that it's solved!

Re DNT, we've reverted the change that turns on DNT by default. Clearly even very clue-d in people that have set DNT get confused about it, and within the industry [DNT is seen as a failed experiment](https://en.wikipedia.org/wiki/Do_Not_Track). We've published version 1.3.5 with these changes. You can still respect DNT by passing the `respect_dnt` option

**Comment by alexshanx** [comment]
@timgl Thanks a lot.
Finally, on July 20 (UTC +8), I upgrade my self-host Posthog Docker image update to date, then I can receive the pageview data again. Everything is ok now.

BTW:
I integration the SDK via npm. 
I have not set the DNT in my browser to true.
